us/en/skymiles/blocks/blade/blade.js

```diff
@@ -19,5 +20,20 @@ export default function decorate(block) {
         }
       }
     });
+    const body = row.querySelector('.blade-body');
```

Comment:
The logic is a bit too aggressive. We should only re-wrap if we actually have the `is-link` case.
For instance, when we have regular inline links, this is removing them (learn more is missing):
<img width="1177" alt="Screenshot 2023-04-05 at 8 49 44 AM" src="https://user-images.githubusercontent.com/1235810/230134958-e8cd47e2-b8b1-4178-adee-ab01b46f78ae.png">

when it should keep them in place:
<img width="1170" alt="Screenshot 2023-04-05 at 8 49 47 AM" src="https://user-images.githubusercontent.com/1235810/230134930-897c5769-1c76-4217-aa27-2d93b1c058ff.png">

In this case, the focus would also move just to the inline link and not the whole blade (or card):
<img width="1169" alt="Screenshot 2023-04-05 at 8 51 26 AM" src="https://user-images.githubusercontent.com/1235810/230135328-9f911f19-5af0-4e05-82c1-8f08427f4983.png">

Reply:
You can probably move the whole logic up into the `try/catch` that adds the `is-link` in the first place.
I'm also wondering if we could maybe just do:
```js
if (a && new URL(a.href).pathname === new URL(a.textContent).pathname) {
  row.classList.add('is-link');
  a.innerHTML = row.innerHTML;
  a.querySelector('a').remove();
  row.innerHTML = a.outerHTML;
}
```